wagtail_airtable/management/commands/import_airtable.py

```diff
@@ -0,0 +1,239 @@
+                    raise CommandError("%r is not recognised as a model name." % label)
+
+                models.append(model)
+
+        created = 0
+        updated = 0
+        skipped = 0
+        for model in models:
+            # Wagtail models require a .save_revision() call when being saved.
+            is_wagtail_model = hasattr(model, 'depth')
+            # Airtable global settings.
+            airtable_settings = settings.AIRTABLE_IMPORT_SETTINGS.get(model._meta.label, {})
+            # Set the unique identifier and serializer.
+            airtable_unique_identifier = airtable_settings.get('AIRTABLE_UNIQUE_IDENTIFIER')
+            model_serializer = self.get_model_serializer(airtable_settings.get('AIRTABLE_SERIALIZER'))
+
+            if type(airtable_unique_identifier) == str:
+                # The unique identifier is a string.
+                # Use it as the Airtable Column name and the Django field name
+                airtable_unique_identifier_column_name = airtable_unique_identifier
+                airtable_unique_identifier_field_name = airtable_unique_identifier
+            elif type(airtable_unique_identifier) == dict:
+                # Unique identifier is a dictionary.
+                # Use the key as the Airtable Column name and the value as the Django Field name.
+                airtable_unique_identifier_column_name, airtable_unique_identifier_field_name = (
+                    list(airtable_unique_identifier.items())[0]
+                )
+
+            # Set the Airtable API client.
+            airtable = Airtable(
+                airtable_settings.get('AIRTABLE_BASE_KEY'),
+                airtable_settings.get('AIRTABLE_TABLE_NAME'),
+                api_key=settings.AIRTABLE_API_KEY,
+            )
+
+            # Get all the airtable records for the specified table.
+            # TODO try/catch this in case of misconfiguration.
+            all_records = airtable.get_all()
+
+            # Loop through every record in the Airtable.
+            for record in all_records:
+                record_id = record['id']
+                record_fields = record['fields']
+                mapped_import_fields = model.map_import_fields(incoming_dict_fields=record_fields)
+                serialized_data = model_serializer(data=mapped_import_fields)
+                serialized_data.is_valid()
+
+                # Look for a record by it's airtable_record_id.
+                # If it exists, update the data.
+                try:
+                    obj = model.objects.get(airtable_record_id=record_id)
+                except model.DoesNotExist:
+                    obj = None
+
+                if obj:
+                    # Model object was found by it's airtable_record_id
+                    if serialized_data.is_valid():
+                        for field_name, value in serialized_data.validated_data.items():
+                            if field_name == "tags":
```

Comment:
No I actually havent even tried working with other m2ms in this manner. But that can all be handled on a case-by-case basis using serializers and methods for importing/exporting values. 

I also thought that having tags in there was an odd case since its so specific. Ill look into checking the field type, that'll support a model/page that has multiple tag fields at least!

Reply:
Alrighty, I finally got around to adding in Tag and m2m field support. Newest update is here: https://github.com/wagtail/wagtail-airtable/blob/initial-development/wagtail_airtable/management/commands/import_airtable.py#L133